tsconfig.schematics.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file should be in the `libs/angular-routing` folder

Reply:
Done.